core/designsystem/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
     id("droidkaigi.primitive.kmp")
     id("droidkaigi.primitive.kmp.ios")
+    id("droidkaigi.primitive.kmp.compose")
```

Comment:
Added dependencies to use the Preview annotation in FontPreview.kt.
https://github.com/DroidKaigi/conference-app-2025/pull/288/files#diff-c6ecfbf4969e2fb11e753e2f495419622f5c56c376936c30a6606ced7ea0e658